src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -110,4 +119,5 @@ export const coreDeprecationProvider: ConfigDeprecationProvider = ({
   dataPathDeprecation,
   rewriteBasePathDeprecation,
   cspRulesDeprecation,
+  mapManifestServiceUrlDeprecation,
```

Comment:
I think you'll want to combine this with an `unusedFromRoot` as well so that Kibana doesn't crash if this config is unused.